guide/additional-info/reactions.md

```diff
@@ -0,0 +1,246 @@
+
+Using `.find()`, your code would look something like this:
+
+```js
+if (message.content === '!react-custom') {
+	const emoji = message.guild.emojis.find('name', 'ayy');
+	message.react(emoji);
+}
+```
+
+Using `.get()`, your code would look something like this:
+
+```js
+if (message.content === '!react-custom') {
+	const emoji = client.emojis.get(config.emojiID);
+	message.react(emoji);
+}
+```
+
+Of course, if you already have the emoji ID, you should just put that directly inside the `.react()` method. But if you want to do other things with the emoji data later on (e.g. display the name or image URL), it's best to retrieve the full emoji object.
+
+### Reacting in order
+
+If you just put one `message.react()` under another, it won't always react in order as is. This is because `.react()` is a Promise, and as such, an asynchronous operation.
+
+```js
+if (message.content === '!fruits') {
+	message.react('🍎');
+	message.react('🍊');
+	message.react('🍇');
+}
+```
+
+![Reaction race condition](/assets/img/DgDVtcv.png)
+
+As you can see, if you leave it like that, it won't display as you really want it to. It was able to react correctly on the first try, but reacts in a different order each time after that.
+
+Luckily, there are two easy solutions to this. The first would be to to chain `.then()`s in the order you want it to display.
+
+```js
+client.on('message', message => {
+	if (message.content === '!fruits') {
+		message.react('🍎')
+			.then(() => message.react('🍊'))
+			.then(() => message.react('🍇'))
+			.catch(() => console.error('One of the emojis failed to react.'));
+	}
+});
+```
+
+The other would be to use the `async`/`await` keywords.
+
+```js
+// notice the `async` keyword
+client.on('message', async message => {
+	if (message.content === '!fruits') {
+		try {
+			await message.react('🍎')
+			await message.react('🍊')
+			await message.react('🍇');
+		}
+		catch (error) {
+			console.error('One of the emojis failed to react.')
+		}
+	}
+});
+```
+
+If you try again with either of the codeblocks above, you'll get the result you originally wanted!
+
+![Emojis reacting in correct order](/assets/img/1IWSJ1C.png)
+
+<p class="tip">If you aren't familiar with Promises or `async`/`await`, you can read more about them on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) or [our guide page on async/await]((/additional-info/async-await)!</p>
+
+## Awaiting reactions
+
+A common use case for reactions in commands is having a user confirm or deny an action, or creating a poll system. Luckily, we actually [already have a guide page that covers this](/creating-your-bot/collectors)! Check out that page if you want a more in-depth explanation. Otherwise, here's a basic example for reference:
+
+```js
+message.react('👍').then(() => message.react('👎'));
```

Comment:
Perhaps suggest as an optimization trick, to use `Promise.all([promises])`, if they do not care about order (and I guess have more than two promises to put). Like in the thumbsup/down reaction sections, I do not think one would care about the order. Doesn't matter if you include this or not, just a thought.